product_set/models/product_set_line.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2015 Anybox S.A.S
+# Copyright 2016-2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class ProductSetLine(models.Model):
+    _name = "product.set.line"
+    _description = "Product set line"
+    _rec_name = "product_id"
+    _order = "sequence"
```

Comment:
For being more complete.

```suggestion
    _order = "product_set_id, sequence, product_id"
```